presto-main/src/test/java/com/facebook/presto/execution/TestNodeScheduler.java

```diff
@@ -250,6 +250,25 @@ public void testTaskCompletion()
         assertEquals(nodeTaskMap.getPartitionedSplitsOnNode(chosenNode), 0);
     }
 
+    @Test
+    public void testSplitCount()
+        throws Exception
+    {
+        MockRemoteTaskFactory remoteTaskFactory = new MockRemoteTaskFactory(remoteTaskExecutor);
+        Node chosenNode = Iterables.get(nodeManager.getActiveDatasourceNodes("foo"), 0);
+        RemoteTask remoteTask = remoteTaskFactory.createTableScanTask(chosenNode, ImmutableList.of(new Split("foo", new TestSplitRemote()), new Split("bar", new TestSplitRemote())));
+        RemoteTask remoteTask2 = remoteTaskFactory.createTableScanTask(chosenNode, ImmutableList.of(new Split("foo2", new TestSplitRemote())));
+        nodeTaskMap.addTask(chosenNode, remoteTask);
```

Comment:
[optional]
Add a vertical space between declaring fields and using them. Thanks to that test will be more readable as test setup and test body get divided.